sdk/test-utils/recorder/README.md

```diff
@@ -1,9 +1,450 @@
+
+```typescript
+skipQueryParams({
+  myQueryParameter: "theValueIWantStoredInMyRecordings",
+  anotherQueryParameter: "theOtherValueIWantStoredInMyRecordings",
+});
+```
+
+### Ever-changing tests
+
+Sometimes, your tests will do requests with unpredictable code.
+The best way to ensure that your ever-changing tests pass on playback mode is to
+use `setReplacements` or `skipQueryParams` to avoid having the changing contents
+as part of your recordings.
+
+### Supporting parallelism
+
+A common issue while running integration tests is that, sometimes two persons
+or machines might try to run the same set of tests against the same resource.
+This is not directly related to the test-utils-recorder package, but if you're
+getting into issues because of concurrent conflicting requests, we understand,
+and we might be able to help by providing you with the following suggestions:
+
+1. Use randomly generated strings as prefixes or suffixes for the resources you
+create.  This will help you, but it will also only work so far, since new
+resources are likely to get accumulated, even if you set code to delete them in
+between tests, since some tests will eventually fail and crash your program.
+2. Set up a separate program as part of your CI to automatically create and
+destroy new resources each time you run a test. It doesn't sound easy, but it
+might be a better solution.  You'll need to make sure to clear resources in
+case this program fails.
+
+These ideas come with their own issue and things to consider, so please take them
+as ideas. We understand that might not be an easy problem to fix.
+
+### Isomorphic tests
+
+The test-utils-recorder does support running in the browser. If you use Karma,
+as long as your karma configuration is correct, your tests should work both on
+NodeJS and in the browsers!
+
+## Troubleshooting
+
+Besides the usual debugging of your code and tests, if you ever encounter a
+problem while recording your tests, make sure to read the output in the
+recordings. If the output is not what you expected, please follow up the
+[contributing](#contributing) guidelines on how to write an issue for us. We'll
+make sure to handle it as soon as we find the time.
+
+## Next steps
+
+The common recorder might not be used yet in each one of our libraries (we're working on it).
+In the mean time, an easy way to find where we're using this package is by going through the following search link:
+<https://github.com/Azure/azure-sdk-for-js/search?q=test-utils-recorder>
+
+## Contributing
+
+This project welcomes contributions and suggestions. Please read the
+[contributing guidelines](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md)
+for detailed information about how to contribute and what to expect while contributing.
+
+
+### Testing
+
+To run our tests, first install the dependencies (with `npm install` or `rush install`),
+then run the unit tests with: `npm run unit-test`.
+
+---
+
+Thank you for your time!
```

Comment:
Instead, you may want to ask for feedback. "We appreciate feedback on how well this tool works for you." and link to a way to send the team feedback.

Thanking them feels a bit awkward.

Reply:
Doing it!